modules/apps/nft-transfer/ibc_module.go

```diff
@@ -75,25 +73,25 @@ func (im IBCModule) OnChanOpenInit(
 	chanCap *capabilitytypes.Capability,
 	counterparty channeltypes.Counterparty,
 	version string,
-) error {
+) (string, error) {
 	if err := ValidateTransferChannelParams(ctx, im.keeper, order, portID, channelID); err != nil {
-		return err
+		return "", err
 	}
 
 	if strings.TrimSpace(version) == "" {
 		version = types.Version
 	}
 
 	if version != types.Version {
-		return sdkerrors.Wrapf(types.ErrInvalidVersion, "got %s, expected %s", version, types.Version)
+		return "", sdkerrors.Wrapf(types.ErrInvalidVersion, "got %s, expected %s", version, types.Version)
 	}
 
 	// Claim channel capability passed back by IBC module
 	if err := im.keeper.ClaimCapability(ctx, chanCap, host.ChannelCapabilityPath(portID, channelID)); err != nil {
-		return err
+		return "", err
 	}
 
-	return nil
+	return "", nil
```

Comment:
See [transfer module for reference](https://github.com/cosmos/ibc-go/blob/v5.0.0-beta1/modules/apps/transfer/ibc_module.go#L92).

```suggestion
	return version, nil
```

Reply:
The current branch depends on cosmos-sdk v0.46.0-rc1, so the interface return parameter definitions are different, let me upgrade it to v0.46.0

Reply:
The current branch is based on the ibc-go v3 version, and the link points to the v5 version, so if I want to modify it, I need to upgrade the current branch to the v5 version. Do I need to resubmit the PR?

Reply:
I am sorry, but I am not sure if I understand what you mean... My comment above is just intended to make sure that the implementation of the NFT app follows the [latest changes that were made to the ICS-721 spec](https://github.com/cosmos/ibc/pull/791) recently.

Reply:
I understand your idea